src/Files.App/Constants.cs

```diff
@@ -147,6 +147,8 @@ public static class LocalSettings
 			public const string UserSettingsFileName = "user_settings.json";
 
 			public const string FileTagSettingsFileName = "filetags.json";
+
+			public const string ActionsSettingsFileName = "user_settings.actions.json";
```

Comment:
Do we need a separate file? I can see a use case for keeping it in user_settings.json to make it easier for users who want to manually modify the file.

Reply:
I thought it will be a large list. If user edits a different property and breaks somewhat, the app is likely to reset all settings and start over from the beginning. The list won't be such a place I guess.


But I will use the same file.

Reply:
Thank you